### PR TITLE
[NAT] import  for `opentelekomcloud_nat_gateway_v2`

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
## Summary of the Pull Request
Added import for opentelekomcloud_nat_gateway_v2
Updated terraform sdk, to fix stateverify error
Updated tests

## PR Checklist

* [x] Refers to: #1972
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.
* [x] Release notes added.

## Acceptance Steps Performed

```

=== RUN   TestAccNatV2Gateway_importBasic
--- PASS: TestAccNatV2Gateway_importBasic (120.77s)

=== RUN   TestAccNatDnat_basic
--- PASS: TestAccNatDnat_basic (137.96s)
=== RUN   TestAccNatDnatRule_withPort
--- PASS: TestAccNatDnatRule_withPort (174.58s)
PASS
=== RUN   TestAccNatDnat_importBasic
--- PASS: TestAccNatDnat_importBasic (155.60s)
PASS
=== RUN   TestAccNatGateway_basic
--- PASS: TestAccNatGateway_basic (136.06s)
PASS
=== RUN   TestAccNatSnatRule_basic
--- PASS: TestAccNatSnatRule_basic (147.72s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

**Review by artem-lifshits — DISMISSED** [review]

**Review by Aloento — DISMISSED** [review]

**Comment by anton-sidelnikov** [comment]
@gtema, failing on go check, something changed? 
2022-10-24 12:56:57.622955 | TASK [ensure-go : Check installed go version]
2022-10-24 12:56:59.685916 | fedora-pod | ok: ERROR (ignored)
2022-10-24 12:56:59.686117 | fedora-pod | {
2022-10-24 12:56:59.686154 | fedora-pod |   "failed_when_result": false,
2022-10-24 12:56:59.686180 | fedora-pod |   "msg": "[Errno 2] No such file or directory: b'go'",
2022-10-24 12:56:59.686202 | fedora-pod |   "rc": 2
2022-10-24 12:56:59.686224 | fedora-pod | }

**Comment by gtema** [comment]
the problem is not go - this is a check whether it is installed before ensuring it is there.

Failure is coming in linter itsef:
```
TASK [golangci-lint : Run golangci-lint]
2022-10-24 14:13:20.120950 | fedora-pod | ERROR
2022-10-24 14:13:20.121157 | fedora-pod | {
2022-10-24 14:13:20.121200 | fedora-pod |   "delta": "0:01:26.206073",
2022-10-24 14:13:20.121226 | fedora-pod |   "end": "2022-10-24 14:13:19.822259",
2022-10-24 14:13:20.121249 | fedora-pod |   "msg": "non-zero return code",
2022-10-24 14:13:20.121270 | fedora-pod |   "rc": -9,
2022-10-24 14:13:20.121291 | fedora-pod |   "start": "2022-10-24 14:11:53.616186"
2022-10-24 14:13:20.121310 | fedora-pod | }
```

Pls check whether there is real linter bug

File: opentelekomcloud/acceptance/nat/import_opentelekomcloud_nat_gateway_v2_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+				Config: testAccNatV2GatewayBasic,
+			},
+			{
+				ResourceName:      resourceGatewayName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
```
Comment (lego963) [inline opentelekomcloud/acceptance/nat/import_opentelekomcloud_nat_gateway_v2_test.go]: Please, add this test in resource test, don't produce more `acceptance_test` files for existing resources

**Review by lego963 — CHANGES_REQUESTED** [review]

**Review by Aloento — DISMISSED** [review]

**Review by Aloento — DISMISSED** [review]

**Comment by anton-sidelnikov** [comment]
=== RUN   TestAccNatGateway_basic
--- PASS: TestAccNatGateway_basic (136.67s)
=== RUN   TestAccNatV2Gateway_importBasic
--- PASS: TestAccNatV2Gateway_importBasic (121.04s)
PASS


Process finished with the exit code 0

=== RUN   TestAccNatDnat_basic
--- PASS: TestAccNatDnat_basic (141.01s)
=== RUN   TestAccNatDnatRule_withPort
--- PASS: TestAccNatDnatRule_withPort (159.62s)

=== RUN   TestAccNatDnat_importBasic
--- PASS: TestAccNatDnat_importBasic (157.32s)
PASS


Process finished with the exit code 0

File: opentelekomcloud/acceptance/nat/import_opentelekomcloud_nat_gateway_v2_test.go (hunk 1 above)
Comment (anton-sidelnikov) [reply opentelekomcloud/acceptance/nat/import_opentelekomcloud_nat_gateway_v2_test.go]: good point

**Review by Aloento — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**